libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxStepDistance.kt

```diff
@@ -18,9 +18,18 @@ class MapboxStepDistance @JvmOverloads constructor(
 ) : AppCompatTextView(context, attrs, defStyleAttr) {
 
```

Comment:
Noticed that we're still using `@JvmOverloads` here https://github.com/mapbox/mapbox-navigation-android/issues/4134